InterGalacticCurrencyConvertor/src/main/java/ProcessorData.java

```diff
@@ -0,0 +1,24 @@
+package main.java;
+
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Stores the user input that needs to be processed.
+ */
+public final class ProcessorData {
+  private final String REMOVE_INBETWEEN_WHITESPACES = "\\s+";
+  private final String mData;
+
+  public ProcessorData(String data) {
+    Objects.requireNonNull(data, "Null String data provided");
+    Pattern pattern = Pattern.compile(REMOVE_INBETWEEN_WHITESPACES);
```

Comment:
Make this pattern a static final member variable?

Reply:
done